docker-compose/README.md

```diff
@@ -0,0 +1,43 @@
+The files in this directory are temporary and will be used to add device-usb-camera into [edgex-compose](https://github.com/edgexfoundry/edgex-compose) in the near future.
```

Comment:
Add TODO to remove these `docker-compose` files once service accepted into EdgeX Foundry and added to compose builder.

Reply:
Added the TODO note. https://github.com/edgexfoundry-holding/device-usb-camera/pull/11/commits/b2d801d418cc0860212bd79b15c013c06d4194d9